crates/weaver_semconv_gen/src/lib.rs

```diff
@@ -325,10 +326,11 @@ impl ResolvedSemconvRegistry {
     /// Resolve semconv registry (possibly from git), and make it available for rendering.
     fn try_from_registry_repo(
         registry_repo: &RegistryRepo,
+        diag_msgs: &mut DiagnosticMessages,
```

Comment:
Nit: this should be called `warnings` or some such to denote that only non-fatal messages will be recorded here.

Reply:
`warnings` may not be the best name, as diagnostic messages can include warnings, advice, or non-fatal errors. Why not stick with diagnostic messages instead?

Reply:
Mostly I just want to denote that it's only non-fatal errors would show up here.